cf-kube-dist/cf-kube-dist.yml

```diff
@@ -1,70 +0,0 @@
-groups: []
-
-resources:
-- name: src
-  type: git
-  source:
-    branch: develop
-    uri: https://github.com/SUSE/scf.git
-
-- name: ci
-  type: git
-  source:
-    uri: https://github.com/SUSE/cf-ci.git
-    branch: master
-    paths:
-    - cf-kube-dist/*
-
-- name: s3.fissile-binary
-  type: s3
-  source:
-    # endpoint: {{s3-endpoint}}
-    access_key_id: {{aws-access-key}}
-    secret_access_key: {{aws-secret-key}}
-    bucket: {{s3-bucket}}
-    regexp: fissile/fissile-(.*)\.g.*linux.amd64\.tgz
-    private: true
-
-- name: s3.stampy-binary
-  type: s3
-  source:
-    access_key_id: {{s3-access-key}}
```

Comment:
`s3` --> `aws`, ditto fixes for secret-key, bucket, endpoint. See also s3.kube-dist.
Working form is seen with s3.certstrap-binary, and s3.fissile-binary above.